manyfews/webapp/converters.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.gis.geos import Polygon
+
+
+class BoundingBoxUrlParameterConverter:
+    """Class to convert GIS coordinates representing a bounding box into a Polygon"""
+
+    regex = r"(\-?[0-9]+(\.?[0-9]+)?,){3}\-?[0-9]+(\.?[0-9]+)?"
+
+    def to_python(self, value):
```

Comment:
sorry, I am a little bit confused, is this function for converting GIS coordinates into python or polygon? maybe adding some comments will be more clear?